backend/dissemination/test_search_resub_tags.py

```diff
@@ -0,0 +1,96 @@
+from django.test import TestCase
+from dissemination.searchlib.search_resub_tags import (
+    build_resub_tag_map,
+    attach_resubmission_tags,
+)
+
+
+class ResubmissionTagTests(TestCase):
+    def test_tag_most_recent(self):
+        rows = [
```

Comment:
Using `baker.make(...)`, it is possible to create some generic `General` objects that have just enough fields that you can then test this code using the actual models/objects we'll see in production. That might be preferable? Happy to pair to flip these over.